tests/dummy/config/optional-features.json

```diff
@@ -1,3 +1,6 @@
 {
-  "jquery-integration": true
-}
+  "application-template-wrapper": false,
+  "default-async-observers": true,
+  "jquery-integration": false,
```

Comment:
I suspect if revert this back to `true`, CI would work again, but ember@4, right? 
If such, I'd suggest to go ahead and disable ember@4 scenarios for now, and re-enable them in scope of the embroider PR.

Reply:
Ok, that does the trick. All set now.